shpc/main/modules/base.py

```diff
@@ -149,6 +152,8 @@ def uninstall(self, name, force=False):
         if os.path.exists(module_dir):
             self.versionfile.write(module_dir)
 
+        return 
+
```

Comment:
That's superfluous, right ?
```suggestion
        return 

```

Reply:
Fixed